Resources/templates/aliases.php.twig

```diff
@@ -1,13 +1,20 @@
 <?php
 
+{% for alias in aliases|slice(0,1) %}
+/**
+ * @file
+ * Drush site-alias configuration for {{ aliases["ac-site"] }}.
```

Comment:
Its only available for ACE, https://github.com/acquia/club/blob/master/src/Command/ACAliasesCommand.php#L119
And site is only avialable for for ACSF
https://github.com/acquia/club/blob/master/src/Command/ACAliasesCommand.php#L155

That being said we could fix that.